spring-cloud-dataflow-shell-core/pom.xml

```diff
@@ -23,6 +23,10 @@
 			<artifactId>spring-cloud-dataflow-rest-client</artifactId>
 		</dependency>
 		<dependency>
+			<groupId>org.springframework.cloud</groupId>
```

Comment:
This dependency may not be needed at the shell core level as it brings in the entire server core as the dependency.